addons/website_sale_comparison/static/src/js/website_sale_comparison.js

```diff
@@ -123,10 +130,48 @@ var ProductComparison = publicWidget.Widget.extend(VariantMixin, {
             });
         } else {
             this.$('.o_comparelist_limit_warning').show();
+            this.el.classList.remove("o_bottom_fixed_element_hidden");
             $('#comparelist .o_product_panel_header').popover('show');
         }
     },
+    _checkProductComparisonButton() {
+        const popoverBottomFixedEl = document.querySelector("#website_cookies_bar .s_popup_bottom");
+        if (!popoverBottomFixedEl) {
+            return;
+        }
+
+        const isNoBackdrop = popoverBottomFixedEl?.classList.contains("s_popup_no_backdrop");
+        const cookieModalDialogEl = popoverBottomFixedEl?.querySelector(".modal-dialog");
+        const isBottomCookie = popoverBottomFixedEl?.classList.contains("o_cookies_classic")
+                            || popoverBottomFixedEl?.classList.contains("o_cookies_discrete");
```

Comment:
When we are already returning from the method, in what other cases `popoverBottomFixedEl` is undefined ?, if there are none such cases we can remove `?`.
Also adapt the methodname based on its functionality `!popoverBottomFixedEl` doesnot make sense here.